scripts/release.sh

```
@@ -0,0 +1,343 @@
+    return 1
+  fi
+  return 0
+}
+
+# Function to compare versions (returns 1 if first version is greater, 0 if equal, -1 if less)
+compare_versions() {
+  local version1=$1
+  local version2=$2
+  
+  # Remove prerelease parts for comparison
+  local v1_stable=$(echo "$version1" | sed -E 's/(-.*)?$//')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)